rita/src/client.rs

```diff
@@ -45,7 +45,7 @@ use actix::registry::SystemService;
 use actix::*;
 use actix_web::*;
 
-use std::sync::{Arc, RwLock};
+use std::sync::{Arc, Mutex, RwLock};
```

Comment:
How's this change significant if there's no change whatsoever to any other part of this file?

Reply:
sorry these were rebase issues from another change